mappings/net/minecraft/util/math/MutableBox.mapping

```diff
@@ -28,23 +28,23 @@ CLASS net/minecraft/class_3341 net/minecraft/util/math/MutableIntBoundingBox
 		ARG 1 other
 	METHOD method_14658 toNbt ()Lnet/minecraft/class_2495;
 	METHOD method_14659 getSize ()Lnet/minecraft/class_2382;
-	METHOD method_14660 getBlockCountX ()I
-	METHOD method_14661 translate (III)V
+	METHOD method_14660 getXSize ()I
+	METHOD method_14661 offset (III)V
 		ARG 1 dx
 		ARG 2 dy
 		ARG 3 dz
 	METHOD method_14662 contains (Lnet/minecraft/class_2382;)Z
 		ARG 1 vec
-	METHOD method_14663 getBlockCountY ()I
-	METHOD method_14664 getBlockCountZ ()I
+	METHOD method_14663 getYSize ()I
+	METHOD method_14664 getZSize ()I
 	METHOD method_14665 empty ()Lnet/minecraft/class_3341;
 	METHOD method_14666 create (IIIIII)Lnet/minecraft/class_3341;
-		ARG 0 x1
-		ARG 1 y1
-		ARG 2 z1
-		ARG 3 x2
-		ARG 4 y2
-		ARG 5 z2
+		ARG 0 minX
+		ARG 1 minY
+		ARG 2 minZ
+		ARG 3 maxX
+		ARG 4 maxY
+		ARG 5 maxZ
```

Comment:
Why was this change to min/max when coords can be given in either position? e.g. `create(1, 1, 1, 2, 2, 2)` and `create(2, 2, 2, 1, 1, 1)` results in same box.
```java
public static MutableIntBoundingBox create(int int_1, int int_2, int int_3, int int_4, int int_5, int int_6) {
    return new MutableIntBoundingBox(Math.min(int_1, int_4), Math.min(int_2, int_5), Math.min(int_3, int_6), Math.max(int_1, int_4), Math.max(int_2, int_5), Math.max(int_3, int_6));
}
```

Reply:
Because when I was working on this the general consensus was against using x/y/z1 and x/y/z2. These names are also not very friendly, and actually rather confusing as I've found in the past.

Reply:
@Sollace I agree with @NinjaPhenix, these should be changed back to `x1`, `x2`. The new names are much more confusing, they're making me think I have to make sure they are min and max, when they can in fact be max and min. Please don't add extra restrictions through parameter names.

> and actually rather confusing as I've found in the past.

What's confusing about being able to make a box between two points, with no restrictions at all...?


@liach @Pyrofab (mentioning you in case like me you weren't aware of this when you reviewed) What are your thoughts?

Reply:
Agreed, at first I thought the `create` method just passed everything to the constructor without changing the argument order.